oas_docs/bundle.json

```
@@ -3622,6 +3622,10 @@
                       "description": "When flapping detection is turned on, alerts that switch quickly between active and recovered states are identified as “flapping” and notifications are reduced.",
                       "nullable": true,
                       "properties": {
+                        "enabled": {
+                          "description": "Indicates whether you want to let the rule's status enter a flapping state.",
```

I made some minor tweaks and described the default configuration. Let me know if this works.

```suggestion
                          "description": "Determines whether the rule can enter to the flapping state. By default, rules are allowed to enter the flapping state.",
```